sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -74,7 +76,7 @@ export class ApplicationAESTController extends BaseController {
   ): Promise<ApplicationSupplementalDataAPIOutDTO> {
     const application = await this.applicationService.getApplicationById(
       applicationId,
-      { loadDynamicData },
+      { loadDynamicData, allowOverwritten: true },
```

Comment:
To ensure the behavior of API to be un-changed I would strongly suggest to add the query param to the API and not as hard-coded. The default value can be set as false.